src/adapter/src/catalog/open.rs

```diff
@@ -296,47 +301,27 @@ impl Catalog {
                 state.create_temporary_schema(&SYSTEM_CONN_ID, MZ_SYSTEM_ROLE_ID)?;
             }
 
+            // Apply updates from the durable catalog.
             let updates = txn.get_updates().collect();
             state.apply_updates_for_bootstrap(updates);
 
-            // This mirrors the `persist_txn_tables` "system var" into the catalog
-            // storage "config" collection so that we can toggle the flag with
-            // Launch Darkly, but use it in boot before Launch Darkly is available.
-            txn.set_persist_txn_tables(state.system_config().persist_txn_tables())?;
-            Catalog::load_builtin_types(&mut state, &mut txn)?;
-
-            let persisted_builtin_ids: BTreeMap<_, _> = txn
-                .get_system_object_mappings()
-                .map(|mapping| (mapping.description, mapping.unique_identifier))
+            // Load all builtin types.
+            let (builtin_types, builtin_non_types): (Vec<_>, Vec<_>) = builtin_item_ids.all_builtins.into_iter().partition(|(builtin, _)| matches!(builtin, Builtin::Type(_)));
```

Comment:
do you mind breaking these onto new lines? unless this is what `rustfmt` automatically does
```suggestion
            let (builtin_types, builtin_non_types): (Vec<_>, Vec<_>) = builtin_item_ids
                .all_builtins
                .into_iter()
                .partition(|(builtin, _)| matches!(builtin, Builtin::Type(_)));
```

Reply:
Done, `rustfmt` seems to be completely broken on this file for some reason. I've noticed a ton of unformatted code that is not fixed by `rustfmt`.